app/views/_volume-claim-templates.html

```diff
@@ -31,14 +31,15 @@
     <div
       row class="pod-template-image icon-row"
       ng-if="template.spec.selector.matchLabels">
-      <div>
+      <div class="icon-wrap">
         <span class="fa fa-tag" aria-hidden="true"></span>
       </div>
       <div flex class="word-break">
         <span class="pod-template-key">Selector:</span>
-        <span ng-repeat="(key, value) in template.spec.selector.matchLabels">
+        <!-- <span ng-repeat="(key, value) in template.spec.selector.matchLabels">
```

Comment:
I believe this block that is commented out can now be removed.  The `<selector selector="template.spec.selector"></selector>` should cover it.